internal/host/transition_test.go

```diff
@@ -286,6 +286,7 @@ var _ = Describe("RegisterHost", func() {
 			Expect(h.Progress.CurrentStage).To(BeEmpty())
 			Expect(h.Progress.ProgressInfo).To(BeEmpty())
 			Expect(h.NtpSources).To(BeEmpty())
+			Expect(h.ImagesStatus).To(BeEmpty())
```

Comment:
Are we adding anything to this in the test implementation?
I don't see anything explicit in the create call.

Reply:
right, added